models/status.js

```diff
@@ -0,0 +1,63 @@
+const firestore = require("../utils/firestore");
+const tagModel = firestore.collection("status");
+
+/**
+ *
+ * @param tagData { Object }: Data of the tag
+ * @returns {Promise<tagModel|Object>}
+ */
+
+const addTag = async (tagData) => {
+  try {
+    tagData.type = tagData.type.toUpperCase();
+    const { id } = await tagModel.add(tagData);
+    return { id, tagData };
+  } catch (err) {
+    logger.error("Error in creating Tag", err);
+    throw err;
+  }
+};
+
+/**
+ *
+ * @param tagid { String }: tagid of the tag
+ * @returns {Promise<tagModel|string>}: returns id of the deleted tag
+ */
+
+const deleteTag = async (tagid) => {
+  try {
+    await tagModel.doc(tagid).delete();
+    return { tagid };
+  } catch (err) {
+    logger.error("Error in deleting Tag", err);
+    throw err;
+  }
+};
+
+/**
+ *
+ * @returns {Promise<tagModel|Array>}: returns an array of all the tags with its meta data
+ */
+
+const getAllTags = async () => {
+  const data = await tagModel.get();
```

Comment:
Add a try-catch block.

Reply:
this was an unwanted file. Pushed by mistake. Thanks